.github/workflows/publish_agentkit.yml

```diff
@@ -0,0 +1,51 @@
+name: Release AgentKit to PyPI
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy-agentkit:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cdp-agentkit-core
+    environment:
+      name: pypi
+      url: https://pypi.org/p/cdp-agentkit-core
```

Comment:
@John-peterson-coinbase - i took a guess here, but let me know!

Reply:
That is correct!

Reply:
sweet!